### PR TITLE
label_smooth

### LINKED ISSUE #22125: label_smooth
#21620

### PR DESCRIPTION
**Pull request by teckno**
closes #22125

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/paddle/nn/functional/common.py [hunk 1]
```diff
@@ -109,3 +109,8 @@ def interpolate(
 def linear(x, weight, bias=None, name=None):
     weight = ivy.swapaxes(weight, -1, -2)
     return ivy.linear(x, weight, bias=bias)
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def label_smooth(label, prior_dist=None, epsilon=0.1, name=None ):
+    return ivy.label_smooth(label, prior_dist, epsilon)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/paddle/nn/functional/common.py:116]: This function isn't present in the ivy API, you will implement a composition

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Check the comments below
Thanks

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**